From 046e421e38f1df69e5a183b663dcb06a8b2bf639 Mon Sep 17 00:00:00 2001
From: Binh Nguyen <binh.nguyen.uw@rvc.renesas.com>
Date: Fri, 13 Jul 2018 11:23:44 +0700
Subject: [PATCH 591/628] Revert "drm: rcar-du: Fix no-retry issue on iwg23s"

This reverts commit b2f1c93cdb23 ("drm: rcar-du: Fix
	no-retry issue on iwg23s")

Signed-off-by: Hung Dong <hung.dong.xd@rvc.renesas.com>
Signed-off-by: Nguyen Van Linh [FGA.BU13] <LinhNV25@fsoft.com.vn>
Signed-off-by: vietn <vietn@fsoft.com.vn>
---
 drivers/gpu/drm/rcar-du/rcar_du_encoder.c | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/drivers/gpu/drm/rcar-du/rcar_du_encoder.c b/drivers/gpu/drm/rcar-du/rcar_du_encoder.c
index a4ef2be..8408359 100644
--- a/drivers/gpu/drm/rcar-du/rcar_du_encoder.c
+++ b/drivers/gpu/drm/rcar-du/rcar_du_encoder.c
@@ -206,9 +206,7 @@ done:
 		devm_kfree(rcdu->dev, renc);
 	}
 
-	if (ret == -EPROBE_DEFER &&
-			(of_machine_is_compatible("iw,iwg20m") ||
-			of_machine_is_compatible("iw,iwg22m")))
+	if (ret == -EPROBE_DEFER)
 		/* Return -EPROBE_DEFER will make kernel trying to initialize
 		 * current output again. But in certain case it causes looping
 		 * and kernel cannot init another output. Disable temporarily */
-- 
2.7.4

